src/TLuaInterpreter.cpp

```diff
@@ -10019,6 +10019,42 @@ int TLuaInterpreter::setDiscordApplicationID(lua_State* L)
     return warnArgumentValue(L, __func__, QStringLiteral("'%1' can not be converted to the expected numeric Discord application id").arg(inputText));
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setDiscordGameUrl
+int TLuaInterpreter::setDiscordGameUrl(lua_State* L)
+{
+    // The invite URL changes what the Discord button opens, and the name is
+    // what it displays on the button. It is not part of rich presence, so it
+    // does not have the API enabled check that those Discord functions need
+    // in order to respect privacy.
+    mudlet* pMudlet = mudlet::self();
+    auto& host = getHostFromLua(L);
+    bool isActiveHost = (pMudlet->mpCurrentActiveHost == &host);
+    int args = lua_gettop(L);
+
+    if (!args) { // no args, blank the invite URL and game name
+        host.setDiscordInviteURL(QString());
+        host.setDiscordGameName(QString());
+        if (isActiveHost) {
+            pMudlet->updateDiscordNamedIcon();
+        }
+        lua_pushboolean(L, true);
+        return 1;
+    }
+    QString inputText = getVerifiedString(L, __func__, 1, "url").trimmed();
+    host.setDiscordInviteURL(inputText.isEmpty() ? QString() : inputText);
+    if (args > 1){
+        inputText = getVerifiedString(L, __func__, 2, "game name").trimmed();
+        host.setDiscordGameName(inputText.isEmpty() ? QString() : inputText);
```

Comment:
Surely the ternary op is redundant? If `inputText.isEmpty()` that is functionally (except for `QString::isNull()` being able to tell the difference) the same as `QString()`? If we are not relying on that small detail then it can be simplified to:
```suggestion
        host.setDiscordGameName(inputText);
```